location1:
  orientation:
    w: 0.4422546311114433
    x: 0.0
    y: 0.0
    z: 0.8968895368218325
  position:
    x: 4.918719994574365
    y: 2.874590281661358
    z: 0.0
location2:
  orientation:
    w: 0.999968191337845
    x: 0.0
    y: 0.0
    z: -0.007975983482875062
  position:
    x: 11.344882564196075
    y: 2.2215430645720917
    z: 0.0
location3:
  orientation:
    w: 0.10161015830332973
    x: 0.0
    y: 0.0
    z: -0.9948242938979588
  position:
    x: 15.734453295296786
    y: 2.859622881238747
    z: 0.0
location4:
  orientation:
    w: 0.9997471248420156
    x: 0.0
    y: 0.0
    z: 0.022487471403504728
  position:
    x: 8.646191032557397
    y: 2.968106997213448
    z: 0.0
location5:
  orientation:
    w: 0.7072710942362725
    x: 0.0
    y: 0.0
    z: -0.7069424299459084
  position:
    x: 4.0521397377470505
    y: 1.1908987510363565
    z: 0.0
location6:
  orientation:
    w: 0.006193395838123594
    x: 0.0
    y: 0.0
    z: -0.9999808207400741
  position:
    x: 3.0932516709783417
    y: 0.03090303023039356
    z: 0.0

location7:
  orientation:
    w: 0.10161015830332973
    x: 0.0
    y: 0.0
    z: -0.9948242938979588
  position:
    x: 15.734453295296786
    y: 2.859622881238747
    z: 0.0
location8:
  orientation:
    w: 0.9997471248420156
    x: 0.0
    y: 0.0
    z: 0.022487471403504728
  position:
    x: 8.646191032557397
    y: 2.968106997213448
    z: 0.0
location9:
  orientation:
    w: 0.7072710942362725
    x: 0.0
    y: 0.0
    z: -0.7069424299459084
  position:
    x: 4.0521397377470505
    y: 1.1908987510363565
    z: 0.0
location10:
  orientation:
    w: 0.006193395838123594
    x: 0.0
    y: 0.0
    z: -0.9999808207400741
  position:
    x: 3.0932516709783417
    y: 0.03090303023039356
    z: 0.0